src/utils.cpp

```diff
@@ -1104,6 +1104,20 @@ FuncsModulesMap parse_traceable_funcs()
                                       ? path_env
                                       : tracefs::available_filter_functions();
 
+  /*
+   * Certain kernel functions are known to cause system stability issues if
+   * traced yet they are not specified in the blacklist so are filtered out
+   * as the list is built. The list of functions have been taken from
```

Comment:
```suggestion
   * Certain kernel functions are known to cause system stability issues if
   * traced (but not marked "notrace" in the kernel) so they should be filtered out
   * as the list is built. The list of functions have been taken from
```

Reply:
We could also mention that as of kernel version 6.9 these functions haven't been marked "notrace" but in the future we could potentially remove them from our internal blocklist if that changes.